src/templates/partials/home-product.hbs

```diff
@@ -10,5 +10,5 @@
       </svg>
     </div>
   </div>
-  <img class="lazy" data-id="{{product.id}}" alt="{{product.name}}" srcset=""></img>
+  <img class="lazy" crossorigin="anonymous" data-id="{{product.id}}" alt="{{product.name}}" srcset=""></img>
```

Comment:
Is this because we're pulling from the CDN?

Reply:
Yes, details here: https://developers.google.com/web/tools/workbox/guides/handle-third-party-requests

Reply:
sgtm